lib/improver/tests/utilities/test_GenerateTimeLaggedEnsemble.py

```diff
@@ -193,19 +192,12 @@ def test_attributes(self):
         expected_attributes = {'institution': 'Met Office'}
         self.assertEqual(result.attributes, expected_attributes)
 
-    @ManageWarnings(record=True)
-    def test_single_cube(self, warning_list=None):
-        """Test only one input cube returns cube unchanged.
-           Also test the warning that is raised."""
+    def test_single_cube(self):
+        """Test only one input cube returns cube unchanged"""
         input_cubelist = iris.cube.CubeList([self.input_cube])
         expected_cube = self.input_cube.copy()
         result = GenerateTimeLaggedEnsemble().process(input_cubelist)
         self.assertEqual(result, expected_cube)
-        warning_msg = "Only a single cube so no differences will be found"
```

Comment:
I'm happy with this change. This warning used to be raised by merge_cubes which is called by this timelagging plugin